tests/selenium/pages/treeherder.py

```diff
@@ -1,9 +1,10 @@
-from pypom import Page
 from selenium.webdriver.common.by import By
 from selenium.webdriver.support import expected_conditions as expected
 
+from pages.base import Base
```

Comment:
This looks like it's a relative import. I think it should be `from .base import Base` instead?

(Python 2 is quite lax about allowing not-technically-correct relative imports - as part of our prep for Python 3 ideally we'll sprinkle some `from __future__ import absolute_import` around, or at least start linting on Python 3)

Reply:
I added `from __future__ import absolute_import` and kept the imports the same, and everything still passed. I then changed them to be relative such as `from .base import Base` without the `__future__` import and they still passed. I've pushed that latest change, but let me know if you would prefer something else.